libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2268,6 +2268,51 @@ MAV_RESULT GCS_MAVLINK::handle_command_set_message_interval(const mavlink_comman
     return MAV_RESULT_FAILED;
 }
 
+MAV_RESULT GCS_MAVLINK::handle_command_get_message_interval(const mavlink_command_long_t &packet)
+{
+    if (comm_get_txspace(chan) < PAYLOAD_SIZE(chan, MESSAGE_INTERVAL) + PAYLOAD_SIZE(chan, COMMAND_ACK)) {
+        return MAV_RESULT_TEMPORARILY_REJECTED;
+    }
+
+    const uint32_t mavlink_id = (uint32_t)packet.param1;
+    if (mavlink_id >= 2 << 15) {
+        // response packet limits range this works against!
+        mavlink_msg_message_interval_send(chan, mavlink_id, 0); // not available
+        return MAV_RESULT_ACCEPTED;
```

Comment:
I guess it makes sense to send an ACCEPTED response instead of REJECTED?  it probably doesn't matter either way I suspect.

Reply:
Yes, at the moment it doesn't really matter.  Given the response could never contain it, I guess it is malformed and should be rejected as you say.